### ISSUE TITLE
Cuda error message

### ISSUE THREAD

**Issue by ontheway16** [CortexFoundation/Cortex_Release#1]
Hello,

I am receiving the following message after execution:

```
$ ./bin/miner -pool_uri miner-cn.cortexlabs.ai:8009 -devicetype gpu -verbosity 5 -deviceid 0
./bin/miner: error while loading shared libraries: libcudart.so.9.0: cannot open shared object file: No such file or directory

$ nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2018 NVIDIA Corporation
Built on Tue_Jun_12_23:07:04_CDT_2018
Cuda compilation tools, release 9.2, V9.2.148
```

**Comment by SiNZeRo** [comment]
could please paste the output of following commands:

`echo #LD_LIBRARY_PATH `
`find -name "libcudart*" /usr`
@ontheway16

**Comment by ontheway16** [comment]
Hello, thanks for the input but problem solved, after Cuda 9.0 installed.
